PennMobile/Dining/Views/TransactionTableViewCell.swift

```diff
@@ -7,3 +7,139 @@
 //
 
 import Foundation
+
+class TransactionTableViewCell: UITableViewCell {
+    
+    static let identifier = "transactionCell"
+    
+    var transaction: Transaction! {
```

Comment:
does this need to be required? seems dangerous, would be better as optional

Reply:
This is perfectly safe sine a cell will never be created without a transaction associated with it.